recording-daemon/mix.c

```diff
@@ -107,6 +107,8 @@ unsigned int mix_get_index(mix_t *mix, void *ptr) {
 
 
 int mix_config(mix_t *mix, const format_t *format) {
+
+
```

Comment:
Please remove these superfluous changes (extra empty lines and commented-out code)